lib/screens/HomeScreen.dart

```diff
@@ -28,8 +32,10 @@ class HomeScreenState extends State<HomeScreen> {
 
   @override
   Widget build(BuildContext context) {
+    TrovUser profile = Provider.of<ProfileManager>(context).profile;
+
     return StreamBuilder<QuerySnapshot>(
-          stream: CloudFirebaseHelper.getLocationsStream(ProfileManager().profile.friends),
+          stream: LocationsManager.getLocationsStream(profile.friends),
```

Comment:
We need to access the `LocationsManager` the same way as `ProfileManager` above. That is, as a provider. This can be added in a different PR but wanted to note it.

Reply:
Done